lua/copilot/extensions/auth.lua

```diff
@@ -1,5 +1,105 @@
+local util = require("copilot.util")
```

Comment:
Could you move auth to the parent dir? Extensions is primarily just extended copilot functionality